src/applications/vaos/new-appointment/redux/actions.js

```diff
@@ -282,12 +277,15 @@ export function checkEligibility({ location, siteId, showModal }) {
     try {
       const loadingStartTime = Date.now();
 
-      const eligibilityData = await getEligibilityData(
```

Comment:
Previously, we would grab eligibility "data" and store that in Redux, then do calculations on whether you were eligible or not in selectors/eligibility.js. Instead, this puts all that logic in the patient service file and stores the results in Redux.